app/views/questionnaire.py

```diff
@@ -92,6 +92,13 @@ def post_block(eq_id, form_type, collection_id, group_id, group_instance, block_
     q_manager = get_questionnaire_manager(g.schema, g.schema_json)
 
     this_location = Location(group_id, group_instance, block_id)
+    if 'action[save_come_back]' in request.form:
```

Comment:
What Ben said, its the current pattern we are having to use for post_household_composition